.ddqa/config.toml

```
@@ -10,6 +10,22 @@ github_team = "agent-metrics-logs"
 github_labels = ["team/agent-metrics-logs"]
 exclude_members = ["olivielpeau"]
 
+[teams."Agent Logs"]
+jira_project = "AGNTLOG"
+jira_issue_type = "Task"
+jira_statuses = ["To Do", "In Progress", "Done"]
+github_team = "agent-metrics-logs"
+github_labels = ["team/agent-logs"]
+exclude_members = ["jszwedko"]
+
+[teams."Agent Metrics"]
+jira_project = "AGTMETRICS"
+jira_issue_type = "Task"
+jira_statuses = ["To Do", "In Progress", "Done"]
+github_team = "agent-metrics-logs"
+github_labels = ["team/agent-metrics"]
+exclude_members = ["jszwedko"]
```

```suggestion
exclude_members = []
```
As Jesse is not part of this team